makefile

```diff
@@ -125,6 +125,15 @@ test-integration: clean-integration
 	curl -v http://localhost:9094/data/refresh # hangs until download and parsing completes
 	./bin/batchsearch -local -threshold 0.95
 
+test: run-test-db
+	time go test ./... && echo $$?
+
+test-verbose: run-test-db
+	time go test -v ./... && echo $$?
+
+run-test-db:
+	docker-compose up -d mysql
+
```

Comment:
Can we remove these? I'm not sure why they were added.

Reply:
These are quality of life make commands for me to help with the testing. If you are opposed to adding these then I'm happy to remove them and have a separate make file for just development for me.

Reply:
Yea, can we remove them? Changes to the build scripts aren't part of adding the UK list.